data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
At least It would make sense to leave only one boundary per segment (to eliminate a possibility of a "miraculous recovery" in the middle of a fully developed avitaminosis). So that instead of:
```
[[ -288, -336 ],
[ -337, -384 ],
[ -385, -720 ]]
```
you could write:
```
[ -288, -337, -385 ]
```